utils/CI/mp_test_executor.sh

```diff
@@ -3,6 +3,9 @@ set -e #Error if any line errors
 set -m #Enable job control
 set -v #Print shell commands as they are read
 
+if [ -f ./wesnoth ]; then client="./wesnoth"; else client="./wesnoth-debug"; fi
+if [ -f ./wesnothd ]; then server="./wesnothd"; else server="./wesnothd-debug"; fi
```

Comment:
Why was this left in if the prefix was removed?

Reply:
I forgot to remove it after.